package.json

```diff
@@ -13,8 +13,8 @@
     "test": "vitest -c vitest.config.ts",
     "release:prepare": "npm run reset:repo && npm ci && npm run lint && npm run build",
     "reset:repo": "git clean -f -X -d",
-    "start:example:server:json": "npm run start:server:json --workspace packages/examples",
-    "start:example:server:python": "npm run start:server:python --workspace packages/examples"
+    "start:examples:server:json": "npm run start:server:json --workspace packages/examples",
+    "start:examples:server:python": "npm run start:server:python --workspace packages/examples"
```

Comment:
Reason for adding `example`, was it needed to distinguish the scripts? Also, some are 'examples' while others are 'example'.

Reply:
It was not aligned with `monaco-languageclient` and I was confused, but logically, it should be singular in the start script and the plural should be used as directory name.

Reply:
Done ⬇️